citesphere/src/main/java/edu/asu/diging/citesphere/web/user/concepts/AddConceptController.java

```diff
@@ -49,12 +51,19 @@ public String show(Model model, Principal principal) {
 
     @RequestMapping(value = "/auth/concepts/add", method = RequestMethod.POST)
     public String post(@Validated @ModelAttribute("conceptForm") CitationConceptForm form, BindingResult result,
-            Model model, Principal principal) {
+            Model model, Principal principal, RedirectAttributes redirectAttributes) {
 
-        if (result.hasErrors()) {
+        if(result.hasErrors()) {
+            for(ObjectError error : result.getAllErrors()) {
+                redirectAttributes.addFlashAttribute("show_alert", true);
+                redirectAttributes.addFlashAttribute("alert_msg", error.getDefaultMessage());
+                redirectAttributes.addFlashAttribute("alert_type", "danger");
```

Comment:
error messages should be shown in the form below the input field they belong to.